includes/class-wc-stripe-customer.php

```
@@ -724,25 +724,49 @@ public function attach_source( $source_id ) {
 
-		if ( ! empty( $response->error ) ) {
 			// It is possible the WC user once was linked to a customer on Stripe
 			// but no longer exists. Instead of failing, lets try to create a
 			// new customer.
-			if ( $this->is_no_such_customer_error( $response->error ) ) {
+			if ( $this->is_no_such_customer_error_message( $error_message ) ) {
 				$this->recreate_customer();
 				return $this->attach_source( $source_id );
-			} elseif ( $this->is_source_already_attached_error( $response->error ) ) {
+			} elseif ( $this->is_source_already_attached_error_message( $error_message ) ) {
 				return WC_Stripe_API::get_payment_method( $source_id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_API' in method 'attach_source'. (undefined)

(StaticAccess)